### PR TITLE
Issue/66

### PR DESCRIPTION
**Pull request by brian-learningpool**

### CODE REVIEW

**Comment by dancgray** [comment]
+1

File: templates/accordion.hbs [hunk 1]
```diff
@@ -13,10 +13,12 @@
                 <div class="accordion-item-body-inner">
                     {{{a11y_text body}}}
                     {{#if _graphic}}
+                    {{#if _graphic.src}}
```
Comment (dennis-learningpool) [inline templates/accordion.hbs]: any risk of an error if _graphic is also null? should it be `{{#if (_graphic && graphic.src) }}`? (dunno if that's possible in handlebars)

Comment (oliverfoster) [inline templates/accordion.hbs]: nah, it's not possible, you have to nest them. the way helpers work doesn't facilitate operators.

null is a falsie, so #if _grahpic when _graphic === null will not execute.

File: templates/accordion.hbs [hunk 2]
```diff
@@ -13,10 +13,12 @@
...
                     <div class="accordion-item-graphic">
-                        <img src="{{_graphic.src}}" alt="{{_graphic.alt}}" tabindex="0">     
+                        <img src="{{_graphic.src}}" alt="{{_graphic.alt}}" tabindex="0">
```
Comment (oliverfoster) [inline templates/accordion.hbs]: could you change alt to aria-label @brian-learningpool  ?

**Comment by hrajotia** [comment]
+1

**Comment by moloko** [comment]
+1

**Comment by oliverfoster** [comment]
+1

**Pull request merged**